src/lib/rustContext.ts

```diff
@@ -123,6 +126,30 @@ export default class RustContext {
     return Promise.reject(emptyExecState())
   }
 
+  // Export a scene to a file.
+  async export(
+    format: DeepPartial<OutputFormat3d>,
+    settings: DeepPartial<Configuration>,
+    toastId: string
+  ): Promise<ModelingAppFile[] | undefined> {
+    await this._checkInstance()
+
+    if (this.ctxInstance) {
```

Comment:
Could we change `_checkInstance()` to return the instance? Then we wouldn't need this check everywhere, and the unreachable error cases could go away.

Reply:
ah dope i did not know how to get it not to be mad but that was what i was trying to do, so like if i return it we will be good?

Reply:
done!